test/unit/presenters/person_presenter_test.rb

```diff
@@ -60,8 +60,8 @@ class PersonPresenterTest < ActionView::TestCase
     @person.stubs(:current_role_appointments).returns([role_appointment_1, role_appointment_2])
     @person.stubs(:translated_locales).returns(%i[en es])
 
-    assert_equal [:en], @presenter.translated_locales
-    refute @presenter.available_in_multiple_languages?
+    assert_equal %i(en es), @presenter.translated_locales
+    assert @presenter.available_in_multiple_languages?
   end
 
   test "is available in multiple languages if person and all current roles are available in the same multiple languages" do
```

Comment:
This test description looks like it could be updated.

Reply:
Also, given that the behaviour is now simpler, you might be able to reduce the number of tests.